src/rp2/locales/es/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,757 @@
+"PO-Revision-Date: 2023-04-27 20:23+0200\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: es\n"
+"Language-Team: es <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.12.1\n"
+
+msgid "<Crypto> In-Out"
+msgstr "<Crypto> Entrada-Salida"
+
+msgid "<Crypto> Tax"
+msgstr "<Crypto> Impuesto"
+
+msgid "Account Balances"
+msgstr "Saldos de Cuenta"
+
+msgid "Accounting Method"
+msgstr "Metodo de contabilidad"
+
+msgid "Acquired"
+msgstr "Adquirido"
+
+msgid "Acquired Balance"
+msgstr "Saldo adquirido"
+
+msgid "Acquired Lot"
+msgstr "Lote adquirido"
+
+msgid "Acquired Lot Fraction"
+msgstr "Fracción de lote adquirido"
+
+msgid "Acquired Lot Fraction %"
+msgstr "Fracción de lote adquirido %"
+
+msgid "Acquired Lot Fraction Description"
+msgstr "Descripción de la fracción del lote adquirido"
+
+msgid "Acquired Lot Spot Price"
+msgstr "Precio al contado del lote adquirido"
+
+msgid "Acquired Lot Timestamp"
+msgstr "Marca de tiempo del lote adquirido"
+
+msgid "Acquired Lot {}"
+msgstr "Lote adquirido {}"
+
+msgid "Acquired Lot {} Amount Fraction"
+msgstr "Lote adquirido {} Cantidad de Fracción"
+
+msgid "Acquired Lot {} Cost Basis"
+msgstr "Lote adquirido {} Base de costo"
+
+msgid "Acquired Lot {} Fee Fraction"
+msgstr "Lote adquirido {} Fracción de comisión"
+
+msgid "Amount"
+msgstr "Cantidad"
+
+msgid "Amount Fraction"
+msgstr "Cantidad de Fracción"
+
+msgid "Amount of given asset held"
+msgstr "Cantidad del activo dado retenido"
+
+msgid "Asset"
+msgstr "Activo"
+
+msgid "Asset - Exchange"
+msgstr "Activo - Intercambio"
+
+msgid "Asset Price Lookup Table"
+msgstr "Tabla de búsqueda de precios de activos"
+
+msgid "Asset holder"
+msgstr "Poseedor de activo"
+
+msgid "Average Price"
+msgstr "Precio medio"
+
+msgid "Average Price Paid Per 1 crypto"
+msgstr "Precio medio pagado por 1 crypto"
+
+msgid "Average price at which the crypto was acquired"
+msgstr "Precio medio al que la criptomoneda fue adquirido"
+
+msgid "Average price at which the crypto was acquired, across all transactions"
+msgstr "Precio medio al que se adquirió la criptomoneda, en todas las transacciones"
+
+msgid "Balance"
+msgstr ""
+
+msgid "Balance of all the BUY and EARN transactions for a given exchange and holder"
+msgstr "Saldo de todas las transacciones de BUY y EARN para un intercambio y titular dados"
+
+msgid "Balance of all the SEND and SOLD transactions for which the given exchange and holder are receiver"
+msgstr "Saldo de todas las transacciones SEND y SOLD para las cuales el intercambio y el titular dados son el receptor"
+
+msgid "Balance of all the SEND and SOLD transactions for which the given exchange and holder are sender"
+msgstr "Saldo de todas las transacciones SEND y SOLD para las cuales el intercambio y el titular dados son el remitente"
+
+msgid "Capital"
+msgstr ""
```

Comment:
Even though the Spanish word is the same as the English one here, I suggest filling the string, in the (unlikely) case the English string is changed. Empty translation strings should be used only for English words that don't have a Spanish translation and are commonly used without translation.